client-ts/package.json

```diff
@@ -23,11 +23,13 @@
   },
   "homepage": "https://github.com/aspnet/SignalR#readme",
   "devDependencies": {
+    "@types/msgpack5": "^3.4.0",
     "browserify": "^13.1.1",
     "del": "^2.2.2",
     "gulp": "^3.9.1",
     "gulp-typescript": "^3.1.3",
     "jasmine": "^2.5.2",
+    "msgpack5": "^3.5.0",
```

Comment:
This package is approved for *use* here. I logged https://github.com/aspnet/SignalR/issues/696 for adding the third party notices file.